build.sbt

```diff
@@ -10,6 +10,7 @@ val scalactic = "org.scalactic" %% "scalactic" % "3.2.10"
 val antlrRuntime = "org.antlr" % "antlr4-runtime" % "4.9.3"
 val sourceCode = "com.lihaoyi" %% "sourcecode" % "0.3.0"
 val mainArgs = "com.lihaoyi" %% "mainargs" % "0.5.1"
+val parralelCollections = "org.scala-lang.modules" %% "scala-parallel-collections" % "1.0.4"
```

Comment:
What is this used for?

Reply:
Intellij might have added it automatically. I will remove